celestia/peers.txt

```diff
@@ -1 +1 @@
-
+5b0d1b8cb6f893426168300f69a82750d4be8729@193.35.57.185:11656
```

Comment:
Please make the peers and gentx separate files, so that your PR looks like this:

`celestia/gentx/validatorname.json // `
`celestia/validatorname.txt //`

Afterwards we will consolidate all the peers from your txt files to peers.txt. Reason we ask is because merging PRs the way they are now is slow and problematic. Thank you for your understanding!

Reply:
Hi @liamsi moved the peer to a separate file.